src/SecureElementReader.App/Views/MainWindow.axaml.cs

```diff
@@ -22,6 +22,8 @@ public MainWindow()
          
         public void HideOverlay() => overlayGrid.ZIndex = -1;
 
+
```

Comment:
Delete empty rows 25 and 26.

Never add unnecessary empty lines of code for three reasons:

1. It adds nothing to the code base
2. It adds noise for the code reviewer
3. The code becomes less readable and maintainable